inc/class_datacache.php

```diff
@@ -687,15 +684,14 @@ function update_statistics()
 		');
 		$topposter = $db->fetch_array($query);
 
-		$query = $db->simple_select('users', 'COUNT(*) AS posters', 'postnum>0');
+		$query = $db->simple_select('users', 'COUNT(uid) AS posters', 'postnum>0');
 		$posters = $db->fetch_field($query, 'posters');
 
 		$statistics = array(
 			'time' => TIME_NOW,
-			'top_forum' => (array)$forum,
 			'top_referrer' => (array)$topreferrer,
 			'top_poster' => (array)$topposter,
-			'posters' => (int)$posters,
```

Comment:
Really? I'm a little concerned about your code modifying code that I implemented just for the sake of it, honestly. Does this means I'm entitled to revert your changes on my next PRs?